cluster/manifests/prometheus-node-exporter/daemonset.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       tolerations:
       - key: CriticalAddonsOnly
         operator: Exists
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
```

Comment:
why?

Reply:
So the corresponding pod would be scheduled, see example at [link](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#how-daemon-pods-are-scheduled).